test-framework/test-suites/integration/tests/ansible/test_stacki_host_info.py

```diff
@@ -0,0 +1,139 @@
+		result = run_ansible_module("stacki_host_info")
+
+		assert result.status == "SUCCESS"
+		assert result.data["changed"] == False
+
+		# The frontend's mac will change each run, so assert it is set then remove it
+		assert result.data["hosts"][0]["interfaces"][0]["mac"]
+		del result.data["hosts"][0]["interfaces"][0]["mac"]
+
+		assert result.data["hosts"] == [
+			{
+				'appliance': 'frontend',
+				'boot': {
+					'action': 'os',
+					'nukecontroller': False,
+					'nukedisks': False
+				},
+				'box': 'frontend',
+				'comment': None,
+				'environment': None,
+				'groups': [],
+				'host': 'frontend-0-0',
+				'installaction': 'default',
+				'interfaces': [{
+					'channel': None,
+					'default': True,
+					'interface': 'eth1',
+					'ip': '192.168.0.2',
+					'module': None,
+					'name': 'frontend-0-0',
+					'network': 'private',
+					'options': None,
+					'vlan': None
+				}],
+				'os': host_os,
+				'osaction': 'default',
+				'rack': '0',
+				'rank': '0',
+				'status': {
+					'ssh': 'up',
+					'state': 'online'
+				}
+			},
+			{
+				'appliance': 'backend',
+				'boot': {
+					'action': 'os',
+					'nukecontroller': False,
+					'nukedisks': False
+				},
+				'box': 'default',
+				'comment': None,
+				'environment': None,
+				'groups': [],
+				'host': 'backend-0-0',
+				'installaction': 'default',
+				'interfaces': [{
+					'channel': None,
+					'default': None,
+					'interface': 'eth0',
+					'ip': None,
+					'mac': None,
+					'module': None,
+					'name': None,
+					'network': None,
+					'options': None,
+					'vlan': None
+				}],
+				'os': host_os,
+				'osaction': 'default',
+				'rack': '0',
+				'rank': '0',
+				'status': {
+					'ssh': None,
+					'state': None
+				}
+			}
+		]
+
+	def test_with_name(self, add_host_with_interface, add_group, host, run_ansible_module, host_os):
+		# Add our backend to a few groups
+		result = host.run('stack add host group backend-0-0 group=test')
+		assert result.rc == 0
+
+		add_group("foo")
+		result = host.run('stack add host group backend-0-0 group=foo')
+		assert result.rc == 0
+
+		# Run out module and check its output
+		result = run_ansible_module("stacki_host_info", name="backend-0-0")
+
+		assert result.status == "SUCCESS"
+		assert result.data["changed"] == False
+
+		assert result.data["hosts"] == [{
+ 			'appliance': 'backend',
+			'boot': {
+				'action': 'os',
+				'nukecontroller': False,
+				'nukedisks': False
+			},
+			'box': 'default',
+			'comment': None,
+			'environment': None,
+			'groups': ['foo', 'test'],
+			'host': 'backend-0-0',
+			'installaction': 'default',
+			'interfaces': [{
+				'channel': None,
+				'default': None,
+				'interface': 'eth0',
+				'ip': None,
+				'mac': None,
+				'module': None,
+				'name': None,
+				'network': None,
+				'options': None,
+				'vlan': None
+			}],
+			'os': host_os,
+			'osaction': 'default',
+			'rack': '0',
+			'rank': '0',
+			'status': {
```

Comment:
Would this test fail on a machine where the tdc-dbs pallet is run?

Reply:
Probably, if the status is changed by the pallet.

Reply:
k, the dbs pallet test-framework for integration tests does run the dbs-pallet, which installs a status plugin for the database / pde.

Reply:
K, test should be fixed, as long as it gets through the pipeline